index.js

```diff
@@ -116,10 +134,14 @@ function fastifyOauth2 (fastify, options, next) {
       tokenRequestParams = {},
       scope,
       generateStateFunction = defaultGenerateStateFunction,
-      checkStateFunction = defaultCheckStateFunction,
+      checkStateFunction = defaultCheckStateFunction.bind({
+        redirectStateCookieName: configured.redirectStateCookieName
```

Comment:
```suggestion
        redirectStateCookieName: configured.redirectStateCookieName || DEFAULT_REDIRECT_STATE_COOKIE_NAME
```

Reply:
oh, ok, I see. Sure I can move that default value from `defaultCheckStateFunction` here, it should effectively do the same